st2common/st2common/openapi.yaml

```diff
@@ -1346,6 +1346,50 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /api/v1/executions/{id}/result:
+    get:
+      operationId: st2api.controllers.v1.actionexecutions:action_execution_raw_result_controller.get
+      x-log-result: false
+      x-submit-metrics: false
+      description: |
+        Get one execution.
+      parameters:
+        - name: id
+          in: path
+          description: Entity id
+          type: string
+          required: true
+        - name: download
+          in: query
+          description: True to force result file download instead of returning it as a response
+          type: boolean
+          default: false
+        - name: compress
+          in: query
+          description: True to compress the response body
+          type: boolean
+          default: false
+        - name: pretty_format
```

Comment:
Out of interest, is the UI going to use the pretty format or the non-pretty format, and does this affect how the response appears in the UI at all?

Reply:
Yeah, UI will use pretty format, but keep in mind that this is only used when displaying results for large executions - we don't display it directly in the UI anymore (since that would freeze the browser. etc.), but we just display link to the API endpoint which returns raw result.

Here is a WIP st2web change - https://github.com/StackStorm/st2web/pull/868.